src/dmd/expressionsem.d

```diff
@@ -3503,16 +3507,19 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
             {
                 exp.f = exp.f.toAliasFunc();
                 TypeFunction tf = cast(TypeFunction)exp.f.type;
-                if (!tf.callMatch(null, exp.arguments))
+                const(char)* failMessage;
+                if (!tf.callMatch(null, exp.arguments, 0, &failMessage))
                 {
                     OutBuffer buf;
                     buf.writeByte('(');
                     argExpTypesToCBuffer(&buf, exp.arguments);
                     buf.writeByte(')');
 
                     //printf("tf = %s, args = %s\n", tf.deco, (*arguments)[0].type.deco);
-                    .error(exp.loc, "`%s%s` is not callable using argument types `%s`", exp.e1.toChars(), parametersTypeToChars(tf.parameters, tf.varargs), buf.peekString());
-
+                    .error(exp.loc, "`%s%s` is not callable using argument types `%s`",
```

Comment:
I'm not seeing any test coverage for this error message.

Reply:
I'm not sure how to hit it - when is a `VarExp` a function declaration?

Reply:
Yeah, I'm trying to figure that out too.  test/fail_compilation/ice11919.d hits it with `doBar`, but that's as far as I've got.  If I come up with more info, I'll let you know.

Reply:
This will hit the scope just outside of the error.

```D
void doBar(T)(T t)  { }

void main()
{
    int x;
    doBar(x);
}
```
But I still can't figure out how to hit the error.  I'm beginning to wonder if it's even possible.

Reply:
I've come close with this: 
```d
void g()
{
    (g(5));
}
```
But the `ve.hasOverloads` test is true, despite `g` not having overloads, so the error branch is not hit.

Reply:
This is clearly a non-trivial problem.  I don't think it should hold up this PR.